### ISSUE TITLE
How to revert back to non-blurred after toast auto dismiss

### ISSUE THREAD

**Issue by degloff** [ionicfirebaseapp/getwidget#117]
This is not a bug but probably more a misunderstanding or a missing documentation/description. I find it nice that we can blur the main screen while displaying a toast. However it is not clear to me how to best revert back to non-blurred display. Here is a small snip that is somehow working but calling a future to reset the variable is kind of a hack. How was the intention of the proper usage?

Note that if we would have a button to let the toast close there is no problem like that. Probably we need a handler in the GFFloatingWidget?

```
  Widget toastWidget(BuildContext context) {
    if (showFloatingToast) {
      return GFToast(
        backgroundColor: Colors.white,
        text: S.of(context).referralLinkCopiedToClipboard,
        textStyle: const TextStyle(color: Colors.black87),
        button: GFButton(
          onPressed: () {
            setState(() {showFloatingToast = false;});
          },
          text: S.of(context).ok,
          type: GFButtonType.transparent,
          color: GFColors.SUCCESS,
        ),
        autoDismiss: true,
      );
    } else
      return Container();
  }

  Widget body(BuildContext context, SignInState signInState) {
    var userData = signInState?.userPrivateData;
    if (userData != null && userData.defaultReferralLink != null) {
      return Container(
        child: Center(
          child: ListView(
            children: <Widget>[
              GFCard(
                boxFit: BoxFit.cover,
                title: GFListTile(
                    title: GFTypography(
                      text: S.of(context).personalReferralLink,
                      type: GFTypographyType.typo4,
                      dividerWidth: 40,
                      dividerColor: Theme.of(context).primaryColor,
                    ),
                    subTitle: Padding(
                      padding: const EdgeInsets.only(top: 8.0),
                      child: Text('Share this link to invite new users.',
                          style: TextStyle(fontSize: 12, color: Colors.black87)),
                    ),
                    icon: GFIconButton(
                      icon: Icon(Icons.content_copy, color: GFColors.DARK, size: 30),
                      type: GFButtonType.transparent,
                      onPressed: () async {
                        Clipboard.setData(ClipboardData(text: userData.defaultReferralLink));
                        setState(() {
                          showFloatingToast = !showFloatingToast;
                          Future.delayed(Duration(milliseconds: 700), () => setState(() {showFloatingToast = false;}));
                        });
                      }
                    )
                ),
                content: qrImage(signInState, 300),
                buttonBar: GFButtonBar(
                  alignment: WrapAlignment.start,
                  children: <Widget>[
                    GFButton(
                      onPressed: () {},
                      text: 'Read More',
                    ),
                  ],
                ),
              ),
            ],
          ),
        ),
      );
    } else {
      return Container(
        child: Center(
          child: Text(S.of(context).couldNotGetReferralLink),
        ),
      );
    }
  }

  @override
  Widget build(BuildContext context) {
    SignInState signInState = Provider.of<SignInState>(context);

    return Scaffold(
      appBar: AppBar(
        title: Text(S.of(context).referrals),
        elevation: 2.0,
      ),
      //drawer: DrawerWidget(),
      body: SafeArea(
        child: GFFloatingWidget(
            verticalPosition: MediaQuery.of(context).size.width * 0.5,
            horizontalPosition: MediaQuery.of(context).size.height * 0.02,
            showblurness: showFloatingToast,
            blurnessColor: Colors.black45,
            child: toastWidget(context),
            body: body(context, signInState)
        ),
        bottom: false,
      ),
    );
  }
```

Thanks for a quick feedback or a suggestion how to use it properly.

**Comment by akshaybhange** [comment]
I'm facing similar kind of issue where toast is dismissed but the background has 30% black overlay that is not getting removed .. here is my code 
```
showDialog(
          context: context,
          builder: (BuildContext context) {
            return StatefulBuilder(builder: ((stateContext, setInnerState) {
              return GFFloatingWidget(
                  child: GFToast(
                    text: 'This item already has the label “travel”',
                    autoDismiss: true,
                  ),);
            }));
          });`
```

**Comment by deepikahr** [comment]
Hi, above mentioned issue is fixed in GetWidget verison 2.0.4. Please go through it.
